crates/types/src/rpc/transactions.rs

```diff
@@ -101,6 +103,7 @@ impl Transaction {
     pub fn create_common_receipt(
         &self,
         transaction_events: &[Event],
+        transaction_messages: &[MessageToL1],
```

Comment:
Due to the argument added by #233, I've now clippy complaining about too many arguments on this function.
What is the strategy you want to adopt? We silent the warning for now? Or do you prefer a rework of the arguments?

Reply:
If you already know how you would rework the arguments, I'd prefer that. Maybe the whole execution_info could be stored in `fn create_accepted`? Or just the `actual_fee`? Or do you have something else in mind?

Reply:
Not sure yet about this one, trying to get the big picture of possible rework.
The first thing I see is that as we already import some `rpc` stuff into `starknet` crate, we may choose to directly pass the `TransactionOutput` instead of passing `events`, `messages` and `actual_fee` separately. This will reduce the number of args by 2.

But then, I was trying to see if it's valuable to relocate the `create_common_receipt` as lot's of arguments are coming the `starknet` transaction type. And `starknet` can import `rpc`, but not the opposite.

WDYT?

Reply:
I attempted in this commit b23e9e3e42910f99eb296d979a7320d8198f66ca to pass the transaction output instead of each individual elements.

If it's fine for now, good. If you prefer a rework by moving the `create_common_receipt`, or any other, let me know. :+1:

Reply:
Fine for now, resolving the conversation